### DISCUSSION TITLE
Optimize powered rails

### DISCUSSION THREAD

**Discussion by Dreeam-qwq** [PurpurMC/Purpur#953] [__Dismissed__] [answered]
Optimize the powered rails to prevent some update suppression and the tps drop.

https://github.com/fxmorin/carpet-fixes/blob/565b903b918ab4ad3defbefc13dd7bdee8aa8d8a/src/main/java/carpetfixes/mixins/optimizations/PoweredRailBlock_fasterMixin.java

**Comment by katekerllenevich** [comment, marked as answer]
This isn't really our forte - its probably better to take this upstream (except they will require real data of performance loss that isn't just "the tps drop)
